app/controllers/admin/site_settings_controller.rb

```diff
@@ -1,10 +1,22 @@
 module Admin
   class SiteSettingsController < ApplicationController
+    # Having to hand craft the form for post to update_maximum_concurrent_scrapers
+    # so doesn't include the authenticity_token. Not a big deal because it's only
+    # accessible by admins anyway
+    skip_before_filter :verify_authenticity_token, only: [:update_maximum_concurrent_scrapers]
```

Comment:
Isn't this vulnerable to cross-site request forgery? i.e. an attacker could get an admin to visit a page elsewhere that could update this setting.